daemon/tests/harness/mocks/mod.rs

```diff
@@ -73,12 +73,26 @@ impl Mocks {
             .returning(|msg| wallet::build_party_params(msg));
     }
 
-    pub async fn mock_monitor_oracle_attestation(&mut self) {
+    pub async fn mock_oracle_attestation(&mut self) {
         self.monitor()
             .await
             .expect_oracle_attestation()
             .return_const(());
     }
+
+    pub async fn mock_monitor_attestation(&mut self) {
+        self.oracle()
+            .await
+            .expect_monitor_attestation()
+            .return_const(());
+    }
+
+    pub async fn mock_start_monitoring(&mut self) {
+        self.monitor()
+            .await
+            .expect_start_monitoring()
+            .return_const(());
+    }
```

Comment:
I think we should include these in `mock_common_empty_handlers` otherwise the test is super noisy.